### ISSUE TITLE
Frequent crashes - possibly related to housekeeping

### ISSUE THREAD

**Issue by teeedubb** [janbar/pvr.mythtv#208]
Hi,

I am experiencing frequent kodi crashes when using the mythtv plugin. I believe this is the case as my backend is on a separate machine and is not always online, so I use kodi for long periods with kodi not connected to the backend and do not experience these crashes. While the crashlogs do not show anything in the stacktrace, there is always an entry in the log, there is always an entry in the log for ** debug <general>: AddOnLog: pvr.mythtv: RunHouseKeeping** near the end of the debug log which makes me think it is related to housekeeping.

Log files here (they're too big for pastebin or github)
Note the one on 24/4 at 11pm occurred when I wasnt using live TV but streaming a movie. The rest happened when watching live TV.
https://www.dropbox.com/sh/ew9l9qsadi9ff0v/AABCXuCrJ742MrM2WvTj-pg4a?dl=0

Not sure how to further debug this, but I can provide more info.

Running MythTV 33 and Kodi 20.1

**Comment by teeedubb** [comment]
So I spent some more time digging into this. Dmesg was throwing network negotiation errors and suggesting to replace the cable. I replaced the cable and swapped from the r8169 driver to the r8168 driver as r8169 had given me issues on another machine and I haven't had a crash for several days, with kodi playing PVR content for over 24hrs straight. Odd that these crashes were prevalent when playing PVR content vs content on the LAN, but I guess it has something do with connection to the backend.

Apologies for the red herring.
